src/main/resources/tailormap-api.yaml

```diff
@@ -53,6 +53,17 @@ components:
 
   schemas:
 
+    UserResponse:
```

Comment:
Is this actually needed? 

I think we could get by using HTTP status 200 or 500 or otherwise 401 with `./statusresponses.yaml#/components/schemas/RedirectResponse`

I'm afraid having more than strictly necessary info could evolve into exposing sensitive data in time

Reply:
See below, I see this is the start for the user information endpoint which we can extend later on. We could return a 401 if the user is not logged in, maybe that is a good idea.